fuzzer/group-panics

```diff
@@ -29,7 +29,7 @@ def test(pattern, release=True):
     return stderr
 
 for panic in panics:
-    pattern = re.match(br'Panic caught\. Pattern: "(.*)"', panic).group(1)
+    pattern = re.match(br'Panic caught\. Pattern: (.*)', panic).group(1)
```

Comment:
This isn't because your change, but I'm getting this error:
```
Traceback (most recent call last):
  File "./group-panics", line 32, in <module>
    pattern = re.match(br'Panic caught\. Pattern: (.*)', panic).group(1)
AttributeError: 'NoneType' object has no attribute 'group'
```
Does `.match(...)` return `None` when no match is found? Should that be possible and if yes, how should we handle it?

Reply:
Yeah, it does when the pattern match fails. This occurs when the output file is non-empty but does not contain any crashes.

I've added a null check to skip the line if it doesn't match on anything, which should cover this problem. Honestly, this would be indicative that a more in-depth refactor might be worth considering, but considering how this PR is suppose to just get it back into a working state, I've decided against a larger change like that.